docs/controllers/services/annotations.md

```diff
@@ -17,7 +17,7 @@ If no custom name is specified, a default name is chosen consisting of the chara
 
 ## service.beta.kubernetes.io/do-loadbalancer-protocol
 
-The default protocol for DigitalOcean Load Balancers. Options are `tcp`, `http`, `https`, and `http2`. Defaults to `tcp`.
+The default protocol for DigitalOcean Load Balancers. Options are `tcp`, `udp`, `http`, `https`, and `http2`. Defaults to `tcp`. Note that if UDP is supported a health check configuration configured against a TCP/HTTP/HTTPS service is still required. In order to use UDP protocol with a Load Balancer, please reach out to support with a ticket to enable it. Note: currently, a port cannot be shared between TCP and UDP due to a [bug in Kubernetes](https://github.com/kubernetes/kubernetes/issues/39188).
```

Comment:
I think we've moved away from supporting/expecting the `udp` string value in the `service.beta.kubernetes.io/do-loadbalancer-protocol` annotation, meaning we should drop it here.

Reply:
I removed udp references from here. Left the health check bit.